### ISSUE TITLE
[powershell] collapsible regions

### ISSUE THREAD

**Issue by adbertram** [Microsoft/vscode#10900]
- VSCode Version: 1.4.0
- OS Version: OSX

Code collapsing works well in PowerShell for most things but it doesn't work for the region tags.
# region
## Stuff here
# endregion

**Comment by sandy081** [comment]
Currently folding is supported only through indentation

**Comment by adbertram** [comment]
Correct thus the feature request.

**Comment by aeschli** [comment]
blocked by #3422

**Comment by kendalf1** [comment]
Collapsable is misspelled in the title - it should be collapsible.  This may help with search.

**Comment by aeschli** [comment]
Thanks @kendalf1, I fixed the title

**Comment by aeschli** [comment]
duplicate of #12146
